### PR TITLE
fix update vault on release

### PR DESCRIPTION
**Pull request by michalbagrowski**
resolve https://github.com/aeternity/aeternity/issues/3836

### CODE REVIEW

**Comment by michalbagrowski** [comment]
@dincho can you take a look, will this solve issue on release that vault was not updated? can this cause other issues?

**Review by venimus — APPROVED** [review]

File: .circleci/config/commands/@deploy.yml [hunk 1]
```diff
@@ -19,5 +19,5 @@ deploy_env:
           export DEPLOY_DOWNTIME=<< parameters.downtime >>
           export PACKAGE=<< parameters.package_path >>
           export DEPLOY_DB_VERSION=$(cat deployment/DB_VERSION)
-
-          cd /infrastructure && make deploy
+          
+          cd /infrastructure && make vault-config-update-${DEPLOY_ENV} deploy
```
Comment (dincho) [inline .circleci/config/commands/@deploy.yml:23]: We should perhaps consider having a PHONY `vault-config-update` target in future

**Review by dincho — CHANGES_REQUESTED** [review]
LGTM itself, however the `node-ci` role does not have permissions to update node configuration. That has to be addressed first.

**Comment by michalbagrowski** [comment]
@dincho https://github.com/aeternity/vault/pull/67

File: .circleci/config/commands/@deploy.yml (hunk 1 above)
Comment (michalbagrowski) [reply .circleci/config/commands/@deploy.yml:23]: https://github.com/orgs/aeternity/projects/3#card-78170842

**Review by mmpetarpeshev — APPROVED** [review]
👍

**Review by dincho — APPROVED** [review]

**Pull request merged**